Formula/c/cog.rb

```diff
@@ -26,7 +26,9 @@ def install
     # Prevent Makefile from running `pip install build` by manually creating wheel.
     # Otherwise it can end up installing binary wheels.
     system python3, "-m", "pip", "wheel", "--verbose", "--no-deps", "--no-binary=:all:", "."
-    (buildpath/"pkg/dockerfile/embed").install buildpath.glob("cog-*.whl").first => "cog.whl"
+
+    cog_wheel = buildpath.glob("cog-*.whl").first
+    (buildpath/"pkg/dockerfile/embed").install cog_wheel => cog_wheel.basename.to_s
```

Comment:
Thanks for the PR, we can just do:

```suggestion
    (buildpath/"pkg/dockerfile/embed").install buildpath.glob("cog-*.whl").first
```

Reply:
Thank you! I had the feeling there was a simpler way of doing this. Updated.